compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -339,7 +319,7 @@ function startPrimaryExpr(Tokenizer tok) returns Expr|err:Syntax {
             LiteralExpr expr = { startPos, endPos, value: () };
             return expr;
         }
-        Expr innerExpr = check parseInnerExpr(tok);
+        Expr innerExpr = check parseExpr(tok);
```

Comment:
Using `innerExpr` no longer seems right

Reply:
fixed (Changed `CallStmt` to avoid type conflicts. I felt `GroupingExpr` having the field named `expr` makes more sense than having it in `CallStmt`)